.azure-pipelines/ci-build.yml

```diff
@@ -21,248 +18,252 @@ variables:
   ProjectFile: '$(Build.SourcesDirectory)/src/Microsoft.Kiota.Cli.Commons/Microsoft.Kiota.Cli.Commons.csproj'
   PublicKey: 0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9
 
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
 
-stages:
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      vmImage: windows-latest
+    stages:
 
-- stage: build
-  jobs:
-    - job: build
-      steps:
-      - checkout: self
-        submodules: true
+    - stage: build
+      jobs:
+        - job: build
+          steps:
+          - checkout: self
+            submodules: true
 
-      - task: UseDotNet@2
-        displayName: 'Use .NET 8'
-        inputs:
-          version: 8.x
+          - task: UseDotNet@2
+            displayName: 'Use .NET 8'
+            inputs:
+              version: 8.x
 
-      - task: PoliCheck@1
-        displayName: 'Run PoliCheck "/src"'
-        inputs:
-          inputType: CmdLine
-          cmdLineArgs: '/F:$(Build.SourcesDirectory)/src /T:9 /Sev:"1|2" /PE:2 /O:poli_result_src.xml'
+          - task: PoliCheck@1

```

Comment:
The steps like PoliCheck/Binskim/CredScan are auto-inject by 1es and should be removed from the pipeline definition. Checkout the PR for a similar change https://github.com/microsoft/kiota-abstractions-dotnet/pull/204/files